### PR TITLE
feat(escalating-issues): Add is escalating analytic

### LINKED ISSUE #48263: [Metrics] Setup Redash dashboard
### Problem Statement

**Objective:**
1. Issue engagement on issues marked unignored today
2. What actions are users taking on issues after they Escalate?
    - Issues Viewed, Archived, Resolved, Deleted, Assigned
    - See this at an org & project & issue level
3. Total # of issue escalations detected at a project/org level, by date (day)

### Solution Brainstorm

**Acceptance Criteria:**

- create a redash dashboard that displays the metrics described above 

### Product Area

Issues

### PR DESCRIPTION
**Pull request by jangjodi**
Add analytic event when an issue starts escalating.
This will allow us to make the necessary dashboards on redash.

WOR-2770
closes https://github.com/getsentry/sentry/issues/48263

### CODE REVIEW

File: src/sentry/issues/analytics.py [hunk 1]
```diff
@@ -7,4 +7,14 @@ class IssueForecastSaved(analytics.Event):
...
+class IssueEscalating(analytics.Event):
+    type = "issue.escalating"
+
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("project_id"),
+        analytics.Attribute("group_id"),
+    )
+
+
```
Comment (barkbarkimashark) [inline src/sentry/issues/analytics.py:19]: Any reason why you're not registering your event?

**Comment by barkbarkimashark** [comment]
nit: is this really a refactor?

File: src/sentry/issues/analytics.py (hunk 1 above)
Comment (jangjodi) [reply src/sentry/issues/analytics.py:19]: thank you, I forgot to do this

Comment (barkbarkimashark) [reply src/sentry/issues/analytics.py:19]: Did you forget to push your commit?

**Review by barkbarkimashark — APPROVED** [review]

File: src/sentry/issues/analytics.py (hunk 1 above)
Comment (jangjodi) [reply src/sentry/issues/analytics.py:19]: yes I did, my bad - thanks again

**Review by armenzg — APPROVED** [review]

**Review by scefali — APPROVED** [review]
@jangjodi looks good, just don't send this to Amplitude

**Comment by jangjodi** [comment]
> @jangjodi looks good, just don't send this to Amplitude

@scefali I just wanted to confirm that nothing in this PR will cause this analytic to be sent to Amplitude? I didn't add it to https://github.com/getsentry/etl/blob/master/etl/operators/analytics_events_schema.py

**Comment by scefali** [comment]
@jangjodi yep you're good

**Pull request merged**